lib/main.js

```diff
@@ -51,13 +51,13 @@ class JavaLanguageClient extends AutoLanguageClient {
         const childProcess = cp.spawn(command, args, { cwd: serverHome })
         this.captureServerErrors(childProcess)
         childProcess.on('exit', exitCode => {
-          if (exitCode == 0 || exitCode == null) {
-          } else {
+          if (childProcess.killed !== true) {
```

Comment:
Much cleaner to use `!childProcess.killed`, right?  Or to have a positive assertion `childProcess.killed` that updates the status bar, and have the else branch handle the failure like before.

Reply:
Status bar should be updated whether or not the process exits cleanly, so I do not think it should be an if/else thing like you mentioned. If your team prefers `!childProcess.killed`, I can update it.

Reply:
Yeah, let's do !childProcess.killed and get this merged. Thanks @jbree !